unittests/linuxliketests.py

```diff
@@ -1920,3 +1920,7 @@ def test_persp_options(self):
         self.check_has_flag(compdb, mainsrc, '-O3')
         self.check_has_flag(compdb, sub1src, '-O2')
         self.check_has_flag(compdb, sub2src, '-O2')
+
+    def test_run_command_external_program(self):
+        testdir = os.path.join(self.unit_test_dir, '125 command list run_command')
+        self.init(testdir, extra_args=['--native-file=' + testdir + '/nativefile.toml'])
```

Comment:
Meson does not use terrible file formats. This is an ini file and is parsed like one. Note file extensions are technically irrelevant.

Reply:
I agree it's technically irrelevant, but don't get the "terrible", and ini doesn't have types like lists or strings so I don't really understand the point. :/

Reply:
The filename is confusing and makes no sense. Meson does not, never has, and never will utilize toml files here.

Why call it nativefile.toml? A better filename would be:

```suggestion
        self.init(testdir, extra_args=['--native-file=' + testdir + '/nativefile.pdf'])
```


OR you could use the conventional filename which is used everywhere in the meson documentation: `nativefile.ini`

> and ini doesn't have types like lists or strings so I don't really understand the point. :/

The value entry for a meson wrap file is an ini string, read by python's configparser module, which is then parsed by the meson.build grammar parser (`mesonbuild.mparser.Parser()`).

All types come from the meson.build syntax. toml types are neither needed nor desirable.

`ini` is used for reading off key=value pairs, which is a role it excels at.